.github/workflows/e2e.yml

```diff
@@ -40,8 +40,15 @@ jobs:
       -
         name: Update daemon.json
         run: |
-          sudo jq '.experimental = true' < /etc/docker/daemon.json > /tmp/docker.json
-          sudo mv /tmp/docker.json /etc/docker/daemon.json
+          if [ ! -f /tmp/foo.txt ]; then
+            # ubuntu 24.04 runners no longer have a default daemon.json present
+            sudo mkdir -p /etc/docker/
+            echo '{"experimental": true}' | sudo tee /etc/docker/daemon.json
+          else
+            # but if there is one; let's patch it to keep other options that may be set.
+            sudo jq '.experimental = true' < /etc/docker/daemon.json > /tmp/docker.json
+            sudo mv /tmp/docker.json /etc/docker/daemon.json
+          fi
```

Comment:
So... hm... it looks like CI was green even if we didn't enable experimental.

I'm wondering now if there's any (e2e) tests that _require_ experimental 🤔 the only "SKIP" I found was in `docker build` to test the `--squash` flag, but I _think_ we no longer gate that option by "experimental" (i.e., experimental only in wording?) https://github.com/docker/cli/blob/591bd174241555b26fb5e92de5d7a9da094523c1/e2e/image/build_test.go#L128

Reply:
Oh; never mind; we do: at least on the _client_ side, but need to check what's on the daemon side https://github.com/docker/cli/blob/591bd174241555b26fb5e92de5d7a9da094523c1/cli/command/image/build.go#L151-L153

Reply:
OK; still is gated on the daemon side https://github.com/moby/moby/blob/9d9488468fe2e08394a8b2fb8ad7fcfba34997f1/api/server/router/build/build_routes.go#L260-L262


However, if that's the _ONLY_ test that differs, perhaps we should combine things; we can always enable experimental on the daemon (we already do).